csrank/constants.py

```diff
@@ -20,25 +20,6 @@
 )
 EXCEPTION_RANKINGS = "Unwanted rankings in {} dataset"
 EXCEPTION_SET_INCLUSION = "Choice Set inclusion/exclusion binary code not present for all the objects in the set."
-allowed_dense_kwargs = [
```

Comment:
Since kwargs to dense layers are now passed by an explicit prefix (`hidden_dense_layer__bias_regularizer`) I assumd that this white list is no longer needed. Do you agree @kiudee?

Reply:
I agree.